pygmt/figure.py

```diff
@@ -235,62 +262,37 @@ def savefig(
         if show:
             launch_external_viewer(fname)
 
-    def show(self, dpi=300, width=500, method="static"):
+    def show(self):
```

Comment:
```suggestion
    def show(self, dpi=300, width=500, method=None):
```
As I just touched on in https://github.com/GenericMappingTools/pygmt/pull/529/files#r578027757 (and also https://github.com/GenericMappingTools/pygmt/pull/529/files#r560667857), maybe keep the dpi/width setting the same for now in this PR, and focus on changing the display `method` to be automated [the suggestion here is to use `display=None` (auto), but this can be overridden].

Reply:
Yes, it sounds a good suggestion.

Reply:
Sorry if this is a lot of work, I do realize you've mostly just adapted Leo's implementation in #529 which changed the `dpi` setting :slightly_smiling_face:

Reply:
Have to re-learn what I did half-year ago 😃